### ISSUE TITLE
Error when trying to preview

### ISSUE THREAD

**Issue by igitur** [tabulapdf/tabula#539]
```
Tabula API version: 1.1.0.160330
Filename: actioncricket_AC_Rule Book 2014 v1.pdf
Internal Server Error (500)


        Request Method:
        POST


        Request URL:
        http://127.0.0.1:8080/pdf/2f9e0b3537df4b4ebba3194e2d22156365b663b8/data

```

The PDF can be found at http://www.actionsports.co.za/downloads/AC_Rule%20Book%202014%20v1.pdf

I suspected there might be problems with that PDF. In fact, I'm unsure whether the text is text or some kind of substitute glyph, because the PDF isn't searchable (for text). Was hoping tabula might help me out here. Anyway, one doesn't expect an exception to be thrown.

**Comment by jeremybmerrill** [comment]
there's a 403 on that document, so I can't test what went wrong. email it to me?

**Comment by igitur** [comment]
[AC_Rule Book 2014 v1.pdf](https://github.com/tabulapdf/tabula/files/438665/AC_Rule.Book.2014.v1.pdf)

**Comment by jeremybmerrill** [comment]
I'm not getting the error you're getting, but the output I get is garbage. I suspect the PDF is broken somehow (as you do). Interestingly, evince (which is a Linux frontend for Libpoppler) handles it correctly. Presumably it's a bug with PDFBox.

**Comment by igitur** [comment]
I'm still getting that error. Using  Tabula 1.1.0-beta.1. I attach the error log from the console. 
[tabula_log.txt](https://github.com/tabulapdf/tabula/files/478674/tabula_log.txt)

**Comment by jeremybmerrill** [comment]
Which page are you trying to extract?

**Comment by igitur** [comment]
Well, any page actually. But if you want a concrete example, try the 3rd page, the Table of Contents.

**Comment by jeremybmerrill** [comment]
I don't get errors, but the output looks like the [this](https://github.com/tabulapdf/tabula/files/604861/tabula-AC_Rule.Book.2014.v1.txt) (which is a bunch of symbols, totally unusable). I still think both problems are likely a PDFBox issue, which'll be fixed when we upgrade to PDFBox 2.0

stacktrace:
```
java.lang.ArrayIndexOutOfBoundsException: 6
	at java.awt.geom.Path2D$Float$TxIterator.currentSegment(Unknown Source)
	at technology.tabula.ObjectExtractor.strokeOrFillPath(ObjectExtractor.java:226)
	at technology.tabula.ObjectExtractor.strokePath(ObjectExtractor.java:275)
	at org.apache.pdfbox.util.operator.pagedrawer.StrokePath.process(StrokePath.java:47)
	at org.apache.pdfbox.util.PDFStreamEngine.processOperator(PDFStreamEngine.java:557)
	at org.apache.pdfbox.util.PDFStreamEngine.processOperator(PDFStreamEngine.java:532)
	at org.apache.pdfbox.util.operator.pagedrawer.FillNonZeroAndStrokePath.process(FillNonZeroAndStrokePath.java:52)
	at org.apache.pdfbox.util.PDFStreamEngine.processOperator(PDFStreamEngine.java:557)
	at org.apache.pdfbox.util.PDFStreamEngine.processOperator(PDFStreamEngine.java:532)
	at org.apache.pdfbox.util.operator.pagedrawer.CloseFillNonZeroAndStrokePath.process(CloseFillNonZeroAndStrokePath.java:47)
	at org.apache.pdfbox.util.PDFStreamEngine.processOperator(PDFStreamEngine.java:557)
	at org.apache.pdfbox.util.PDFStreamEngine.processSubStream(PDFStreamEngine.java:268)
	at org.apache.pdfbox.util.PDFStreamEngine.processSubStream(PDFStreamEngine.java:235)
	at org.apache.pdfbox.util.PDFStreamEngine.processStream(PDFStreamEngine.java:215)
	at technology.tabula.ObjectExtractor.drawPage(ObjectExtractor.java:153)
	at technology.tabula.ObjectExtractor.extractPage(ObjectExtractor.java:108)
```
